docs/vectorizer-quick-start-openai.md

```diff
@@ -32,7 +32,6 @@ On your local machine:
         image: timescale/pgai-vectorizer-worker:v0.2.1
         environment:
           PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
-          OPENAI_API_KEY: <your-api-key>
```

Comment:
While this will work with the new version, this is actually going to be confusing until we push a new release. Unless we change this to refer to main for the vectorizer worker? Is that image kept up to date on each `main` push?

Reply:
yeah we should wait to merge this until this is resolved

Reply:
I'll put this in a separate PR